Sources/Plasma/NucleusLib/pnInputCore/plInputMap.cpp

```diff
@@ -47,6 +47,10 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 
 #include "plResMgr/plLocalization.h"
 
+#ifdef HS_BUILD_FOR_APPLE
+#include <Carbon/Carbon.h>
```

Comment:
This is a super nit-picky distinction, but if this is only available on macOS (and not other Apple platforms) it should be guarded by `HS_BUILD_FOR_MACOS` rather than `HS_BUILD_FOR_APPLE`

Reply:
At some point this should be transitioned over to the Game Controller framework which is shared between iOS and macOS. Game Controller didn't add keyboard support until macOS 11 though.